packages/@okta/i18n/package.json

```diff
@@ -1,30 +1,31 @@
 {
   "_args": [
     [
-      "@okta/i18n@34.3.0",
+      "@okta/i18n@34.5.0",
       "/home/vagrant/okta/okta-signin-widget"
     ]
   ],
-  "_from": "@okta/i18n@34.3.0",
-  "_id": "@okta/i18n@34.3.0",
+  "_from": "@okta/i18n@34.5.0",
+  "_id": "@okta/i18n@34.5.0",
   "_inCache": true,
+  "_installable": true,
   "_location": "/@okta/i18n",
   "_phantomChildren": {},
   "_requested": {
     "name": "@okta/i18n",
-    "raw": "@okta/i18n@34.3.0",
-    "rawSpec": "34.3.0",
+    "raw": "@okta/i18n@34.5.0",
+    "rawSpec": "34.5.0",
     "scope": "@okta",
-    "spec": "34.3.0",
+    "spec": "34.5.0",
     "type": "version"
   },
   "_requiredBy": [
-    "/"
+    "#USER"
```

Comment:
I have seen this in a few PRs, but if I remember correctly this is not needed, it should be ```/```